app/retail/templates/about.html

```diff
@@ -99,60 +132,91 @@ <h6 style="text-align: center;" >{% trans "Who, what, when, where, and most impo
-              {% blocktrans %}<a href="http://linkedin.com/in/owocki/">Kevin Owocki</a> has a BS in Computer Science, 15 years experience in Open Source Software and Technology Startups. He is a volunteer in the Boulder Community for several community organizations, and an avid open source developer. His work has been featured in <a href="http://techcrunch.com/2011/02/10/group-dating-startup-ignighter-s-3-million/" >TechCrunch</a>, <a href="http://www.cnn.com/2011/BUSINESS/03/29/india.online.matchmaking/" >CNN</a>, <a href="http://www.inc.com/30under30/2011/profile-adam-sachs-kevin-owocki-and-dan-osit-founders-ignighter.html" >Inc Magazine</a>, <a href="http://www.nytimes.com/2011/02/20/business/20ignite.html?_r=4&amp;amp;pagewanted=1&amp;amp;ref=business"  >The New York Times</a>, <a href="http://boingboing.net/2011/09/24/tosamend-turn-all-online-i-agree-buttons-into-negotiations.html" >BoingBoing</a>, <a href="http://www.wired.com/2015/12/kevin-owocki-adblock-to-bitcoin/"  >WIRED</a>, <a href="https://www.forbes.com/sites/amycastor/2017/08/31/toothpick-takes-top-prize-in-silicon-beach-ethereum-hackathon/#6bf23b7452ad" >Forbes</a>, and <a href="http://www.techdigest.tv/2007/08/super_mario_get_1.html" >TechDigest</a>.{% endblocktrans %}
-            </p>
+          <div class="teamProfileLeft">
+            <img class="teamProfileAvatar img-fluid" src="{% static "v2/images/team/justin-bean.jpg" %}">
+            <p><span class="font-bigger-1 font-weight-300"><strong>Justin Bean</strong></span><br><span class="font-smaller-1"><em>Engineering</em></span></p>
+            <a href="https://gitcoin.co/profile/StareIntoTheBeard"><img class="teamProfileSocialImg" src="{% static "v2/images/social/gitcoin.png" %}"></a>
+            <a href="https://github.com/StareIntoTheBeard"><img class="teamProfileSocialImg" src="{% static "v2/images/social/github.png" %}"></a>
+            <p><strong>Favorite Gitcoin Feature:</strong> Issue Explorer<br><strong>Favorite Food:</strong> Sushi</p>
           </div>
         </div>
-        <div class="col p-5 animated fadeInRightShort ">
-          <div class="row">
-            <h4 class="pt-1">{% trans "Post Launch (2017+)" %}</h4>
+        <div class="col pb-4 animated text-center fadeInRightShort">
+          <div class="teamProfileRight">
+            <img class="teamProfileAvatar img-fluid" src="{% static "v2/images/team/mark-beacom.jpg" %}">
+            <p><span class="font-bigger-1 font-weight-300"><strong>Mark Beacom</strong></span><br><span class="font-smaller-1"><em>Engineering</em></span></p>
+            <a href="https://gitcoin.co/profile/mbeacom"><img class="teamProfileSocialImg" src="{% static "v2/images/social/gitcoin.png" %}"></a>
+            <a href="https://github.com/mbeacom"><img class="teamProfileSocialImg" src="{% static "v2/images/social/github.png" %}"></a>
+            <p><strong>Favorite Gitcoin Feature:</strong> Start/Stop Work<br><strong>Favorite Food:</strong> Dolsot Bibimbap</p>
```

Comment:
@mbeacom Thoughts on breaking this into multiple lines ? (for readability )

Reply:
@thelostone-mc I agree 👍